qemu/tests/cfg/uefi_check_resolution.cfg

```diff
@@ -0,0 +1,18 @@
+- uefi_check_resolution:
+    only q35
+    only ovmf
+    only Linux
```

Comment:
I think the case is not only support Linux.

Reply:
Yes,this case not cares guest OS.Thanks!